### ISSUE TITLE
Admin: Create a demo site

### ISSUE THREAD

**Issue by pjhampton** [pjhampton/Gramophone#76]
We should create a demo site so people can use the theme live to see what its like.

**Comment by DigitalDesignerOnline** [comment]
I would be happy to assist here. I will get started right away.

**Comment by DigitalDesignerOnline** [comment]
any specific criteria you had in mind for highlighting purposes?

**Comment by RickBakkr** [comment]
I'd be happy to assist here as well. I guess multiple online demo's might be a good idea? 
Off-topic: cool, my idea being turned into an issue :-)

**Comment by harry-7** [comment]
someone working on this? If not, can @pjhampton let me know what needs to be done ? May be like a checklist or something?

**Comment by RickBakkr** [comment]
@harry-7 Is that really _that_ useful for a template that really acts as a boilerplate to build your own from?

**Comment by harry-7** [comment]
Why not? Whenever I browse themes for wordpress, I always look at the live demo if it is available than the screenshots provided. Live demo allows me to check various features like responsiveness, etc etc.
So why not have a live demo?

**Comment by pjhampton** [comment]
Hey guys,

Yeah, I feel a live demo of the theme is a good idea because it will get more people using it I reckon, but I agree with @RickBakkr that it's probably not worth too much effort. Anyway, this issue is here for me to do something - but I'm so lazy 😆 

I'll do it one day - I promise

anyway - I will be creating more issues this weekend 👍

**Comment by RickBakkr** [comment]
@pjhampton I love how this specific repository has been surprisingly active during Hacktoberfest. Eager to contribute here this Hacktoberfest too - I believe I have in both previous years :p

**Comment by pjhampton** [comment]
@RickBakkr contributions are not too much higher than other times in the year - https://github.com/pjhampton/Gramophone/graphs/contributors - but if I can get extra help I would love to take it!

Thanks. Maybe I will create a dev group so you guys can be notified on github

**Comment by harry-7** [comment]
@pjhampton yeah please create more :p I like your repo. Seriously, hacktoberfest is the time I get to know new repos.
